com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -618,11 +908,11 @@ def cancel_job(self, job_id=None, job_name=None):
         """Cancel a running job.
 
         Args:
-            job_id(str): Identifier of job to be canceled.
-            job_name(str): Name of job to be canceled.
+            job_id (str, optional): Identifier of job to be canceled.
```

Comment:
Is this 'optional' a standard approach. I thought the fact that the method signature showed it was optional was sufficient? Just not seen it before.